Source/UniformGradient.swift

```diff
@@ -2,12 +2,19 @@ import UIKit
 
 public struct UniformGradient: Gradient {
 
+    public var angle: GradientAngle?
```

Comment:
This seems like a good addition, but I'm not sure it should be optional. If we're going to apply the concept of angles to all the gradients, then I think it should be a required property (and `let` rather than `var`). What do you think?